device/dhcp/clients/dhcpclient.yaml

```diff
@@ -0,0 +1,20 @@
+components:
+  schemas:
+    Dhcp.client:
+      description: >-
+        Configuration for emulated DHCPv4 or DHCPv6 Client on a single Interface.
+        https://www.rfc-editor.org/rfc/rfc2131.html
+      type: object
+      x-status:
+        status: under_review
+      properties:
+        v4:
+          description: |-
```

Comment:
This is another use case found in IxNetwork sample called `DHCP over PPPoE`. 
![image](https://github.com/open-traffic-generator/models/assets/35574510/2bf7a8fd-2dab-471b-9250-0fbc10e01577)


This is probably not a use case right now. But I feel our model will not extend it to accommodate this use case.